apps/src/templates/teacherDashboard/shapes.jsx

```diff
@@ -42,4 +42,4 @@ export const loadErrorShape = PropTypes.shape({
   message: PropTypes.string.isRequired,
 });
 
-export const OAuthSectionTypes = makeEnum('google_classroom', 'clever');
+export const OAuthSectionTypes = makeEnum('google_classroom', 'clever', 'microsoft_classroom');
```

Comment:
+cc @poorvasingal: Are we expecting the new Section flow to ship with Microsoft Classroom?  Head's up this isn’t on my radar at all.